### PR TITLE
[14.0] interest on questions and answers

### PR DESCRIPTION
**Pull request by marielejeune**

### CODE REVIEW

**Comment by marielejeune** [comment]
@qgroulard review requested

File: mozaik_petition/views/petition_petition.xml [hunk 1]
```diff
@@ -178,6 +170,11 @@
                                                     name="is_mandatory"
                                                     attrs="{'invisible': [('question_type', '!=', 'tickbox')]}"
                                                 />
+                                               <field
+                                                    name="interest_ids"
+                                                    widget="many2many_tags"
+                                                    attrs="{'invisible': [('question_type', '!=', 'tickbox')]}"
```
Comment (qgroulard) [inline mozaik_petition/views/petition_petition.xml:176]: Isn't the same for open (text) questions ?

File: mozaik_event_question_thesaurus/views/event_event.xml [hunk 2]
```diff
@@ -8,15 +8,15 @@
         <field name="model">event.event</field>
         <field name="inherit_id" ref="website_event_questions.event_event_view_form" />
         <field name="arch" type="xml">
+<!--            <xpath-->
```
Comment (qgroulard) [inline mozaik_event_question_thesaurus/views/event_event.xml]: Please remove this kind of commented code.

File: mozaik_event_tickbox_question/views/event_event.xml [hunk 3]
```diff
@@ -25,6 +25,11 @@
                     name="is_mandatory"
                     attrs="{'invisible': [('question_type', '!=', 'tickbox')]}"
                 />
+                <field
+                    name="interest_ids"
+                    widget="many2many_tags"
+                    attrs="{'invisible': [('question_type', '!=', 'tickbox')]}"
```
Comment (qgroulard) [inline mozaik_event_tickbox_question/views/event_event.xml]: Same as in petition, don't you need this also for open questions ?

File: mozaik_event_tickbox_question/models/event_question.py [hunk 4]
```diff
@@ -17,6 +17,7 @@ class EventQuestion(models.Model):
         "the box to continue the registration.",
         default=False,
     )
+    interest_ids = fields.Many2many("thesaurus.term", string="Interests")
```
Comment (qgroulard) [inline mozaik_event_tickbox_question/models/event_question.py]: You could want interest_ids at question level for other types than tickbox thus I'm not sure mozaik_event_tickbox_questions is the right place for this code.

**Pull request merged**